### DISCUSSION TITLE
[374040] Portal Knights 传送门骑士 [本体]

### DISCUSSION THREAD

**Discussion by Astraios-A** [pjy612/SteamManifestCache#401] [Share]
`appid:374040`
keys:
```
374041;a50a492cd588398e0b8d7827635a4331eaa6c7a626722137840b655e532dd725
```
[374040 Portal Knights.zip](https://github.com/pjy612/SteamManifestCache/files/13993598/374040.Portal.Knights.zip)
Email:xiaruoxing1006@163.com
Code:C24D4A3120E0E0AD

**Comment by pjy612** [comment]
* 密钥不对
* 无清单

**Reply by Astraios-A** [reply]
已追加修改，烦请确认
